### ISSUE TITLE
pygame 2.1.0 and pygame_gui 0.5.7 incompatible on raspberry pi os

### ISSUE THREAD

**Issue by jonbobcar** [MyreMylar/pygame_gui#221]
**Describe the bug**
I'm trying to use pygame_gui on a raspberry pi zero. I'm using the previous release of Pi OS (buster) due to an incompatibility between the new OS and the screen I have connected to the Pi. Importing pygame and importing pygame_gui produce the error shown in the pastebin below (schreenshots section). Downgrading to pygame 1.9.6 fixes the issue.

**To Reproduce**
Steps to reproduce the behaviour:
1. fresh install of raspberry pi os
2. install pygame_gui and update pygame to 2.1.0
3. open python3 and import pygame_gui

**Expected behaviour**
pygame and pygame_gui should import.

**Screenshots**
https://pastebin.com/raw/XkZRdit5

**Platform and software (please complete the following information):**
 - OS: Raspberry Pi OS Buster 2021-05-07
 - Pygame GUI version 0.5.7
 - Pygame version 2.10
 - Python 3.7.3

**Additional context**

**Comment by MyreMylar** [comment]
That looks like an error during compilation/install of pygame 2.1.0 to me. I think you don't have SDL 2 or maybe just not sdl_ttf on your pi. Pygame's font submodule depends on some libraries that it looks like your pi didn't have or couldn't find during installation (specifically libSDL2_ttf-2.0).

I'm not a raspberry pi/linux expert these days, I think if you are going to build the latest pygame on pi you also need to install a recent sdl2 and sdl2-ttf before trying to build pygame. 

try this: 
```
sudo apt install libasyncns0 libflac8 libfluidsynth1 libice6 libjack0 libjbig0 libmodplug1 libmpg123-0 libogg0 libopus0 libopusfile0 libportmidi0 libpulse0 libsdl2-2.0-0 libsdl2-image-2.0-0 libsdl2-mixer-2.0-0 libsdl2-ttf-2.0-0 libsm6 libsndfile1 libsndio7.0 libtiff5 libvorbis0a libvorbisenc2 libvorbisfile3 libwayland-client0 libwayland-cursor0 libwayland-egl1 libwebp6 libx11-xcb1 libxcursor1 libxfixes3 libxi6 libxinerama1 libxkbcommon0 libxrandr2 libxrender1 libxss1 libxtst6 libxxf86vm1
```

and maybe pip installing pygame again?

**Comment by jonbobcar** [comment]
Hey, thanks. You were correct in that your string of apt installs fixed the issue.

Raspberry Pi OS comes with Pygame installed, so there is some dependency broken in Pygame 2.1.0 on Raspberry Pi.

Previously I tried with Buster (previous release of Raspberry Pi OS) which comes with Pygame 1.9.6 installed and it worked until I upgraded to Pygame 2.1.0.

Then I tried with Bullseye (newest release of Raspberry Pi OS) which comes with Pygame 2.1.0 installed and Pygame GUI did not work.

Anyway, thanks again for the help!
